webcompat/webhooks/helpers.py

```diff
@@ -30,7 +32,63 @@ def new_opened_issue(payload):
 
 def handle_type_media(payload):
     """Checks duplicity when webhook receive a type-media issue."""
-    pass
+    media_issues = get_media_issues(MEDIA_PATH)
+    known_media, issue = is_known_media(payload, media_issues)
+    # Wondering if I should specialize the different requests
+    # (or not) like in set_labels.
+    headers = {
+        'Authorization': 'token {0}'.format(app.config['OAUTH_TOKEN'])
+    }
+    if known_media:
+        # 1. Remove label status-needstriage
+        # DELETE /repos/:owner/:repo/issues/:number/labels/:name
+        # delete_labels(['status-needstriage'], issue['number'])
+        path = 'repos/{0}/{1}/labels/status-needstriage'.format(
```

Comment:
so yes most of this part of the code, I stopped. Basically exactly what you are saying, I was exactly ready to do that when I realized that I was about to translate part of GitHub API into methods. If we start to do that we could create a `github.py` and make the methods explicits. We use certain actions at plenty of places in the code. Larger than the webhooks. We didn't really need it because we were mostly doing one type of things. 

TO_DISCUSS in San Francisco.